### PR TITLE
DOC-6644: Add advisories to htmltest

### PR DESCRIPTION
**Pull request by nickvigilante**
Fixes DOC-6644

Also excludes unsupported versions from htmltest check

### CODE REVIEW

File: advisories/a88042.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ This public issue is tracked by [#87305](https://github.com/cockroachdb/cockroac
 
 Users of CockroachDB [v22.1.0](../releases/v22.1.html#v22-1-0) to [v22.1.8](../releases/v22.1.html#v22-1-8) are encouraged to upgrade to [v22.1.9](../releases/v22.1.html#v22-1-9) or a later version.
 
-Users of CockroachDB [v21.2.0](../releases/v22.1.html#v21-2-0) to [v21.2.16](../releases/v22.1.html#v21-2-16) are encouraged to upgrade to [v21.2.17](../releases/v22.1.html#v21-2-17) or a later version.
+Users of CockroachDB [v21.2.0](../releases/v21.2.html#v21-2-0) to [v21.2.16](../releases/v21.2.html#v21-2-16) are encouraged to upgrade to [v21.2.17](../releases/v21.2.html#v21-2-17) or a later version.
```
Comment (andf-crl) [inline advisories/a88042.md:45]: Oh wow, nice catch Nick! This was clearly copy&pasted!

File: advisories/c20230118.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ All users assigned the Developer role in a CockroachDB Cloud organization will n
 - `CONTROLJOB`
 - Cluster node metadata
 
-See [Role Options](../{{site.current_cloud_version}}/alter-user.html#role-options) for more information on these roles.
+See [Role Options](../{{site.current_cloud_version}}/alter-user.html#{% if site.current_cloud_version == "v22.1" %}parameters{% else %}role-options{% endif %}) for more information on these roles.
```
Comment (andf-crl) [inline advisories/c20230118.md:32]: #role-options also present in [v22.2](https://www.cockroachlabs.com/docs/stable/alter-user.html#role-options) though (and all future).

File: advisories/a90146.md [hunk 3]
```diff
@@ -12,9 +12,9 @@ Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
...
 
-In CockroachDB [v22.1.6](../releases/v22.1.html#v22-1-6) to [v22.1.9](../releases/v22.1.html#v22-1-9), changefeeds using the [`initial_scan='only'`](../v22.1/create-changefeed#initial-scan) option can complete with a `successful` status even if not all messages have been emitted to the sink. This occurs consistently unless the table is very small.
+In CockroachDB [v22.1.6](../releases/v22.1.html#v22-1-6) to [v22.1.9](../releases/v22.1.html#v22-1-9), changefeeds using the [`initial_scan='only'`](../v22.1/create-changefeed.html#initial-scan) option can complete with a `successful` status even if not all messages have been emitted to the sink. This occurs consistently unless the table is very small.
 
-The same issue may occur if a changefeed with [`schema_change_policy='stop'`](../v22.1/create-changefeed#options) encounters a schema change. The changefeed may not emit any changes that occurred prior to the schema change being emitted.
+The same issue may occur if a changefeed with [`schema_change_policy='stop'`](../v22.1/create-changefeed.html#options) encounters a schema change. The changefeed may not emit any changes that occurred prior to the schema change being emitted.
```
Comment (andf-crl) [inline advisories/a90146.md:17]: Oh wow again, nice catch!!

File: advisories/a88993.md [hunk 4]
```diff
@@ -37,7 +37,7 @@ Users of CockroachDB [v22.1.0](../releases/v22.1.html#v22-1-0) to [v22.1.8](../r
 - A computed column expression (e.g., `b INT AS (a + 10) STORED` given column a has filter `WHERE a IN (1, 3)`)
 - A `PARTITION BY` clause (e.g., `INDEX (a, ...) PARTITION BY LIST (a) (PARTITION p VALUES ((1), (3)))`)
 
-If the plan shown by [`EXPLAIN (OPT)`](/docs/{{site.versions["stable"]}}/explain#opt-option) uses the multi-column index and shows a union-all, these queries may produce incorrect results.
+If the plan shown by [`EXPLAIN (OPT)`](/docs/{{site.versions["stable"]}}/explain.html#opt-option) uses the multi-column index and shows a union-all, these queries may produce incorrect results.
```
Comment (andf-crl) [inline advisories/a88993.md:40]: Another! Nice one!

**Review by andf-crl — APPROVED** [review]
Nice work!! Great catches on some likely-longstanding errors. Never even seen `../stable/` in the wild!

Small note on `#role-options` for your considerations, but otherwise LGTM. Thanks for the assist!!

File: advisories/c20230118.md (hunk 2 above)
Comment (nickvigilante) [reply advisories/c20230118.md:32]: Yep, but the issue is `current_cloud_version` is listed as v22.1, where the heading doesn't exist. I can remove this block and replace it with `#role-options` once that changes.

Comment (andf-crl) [reply advisories/c20230118.md:32]: Sounds good, but it'd be another thing to remember that's on you if you want to shoulder it. I don't suppose I see another way though, you're right.

**Pull request merged**